.github/configs/labeler.yml

```diff
@@ -0,0 +1,17 @@
+version: 1
+labels:
+  - label: "xsmall-diff"
```

Comment:
Remove the `-diff` suffix from the labels. The size alone is enough to indicate that this is an N size change.

Reply:
Actually, please ensure these match the following examples from how Kubernetes repos do it:
![image](https://user-images.githubusercontent.com/101085/215348969-dcecc193-5f52-4a79-b809-80c724d0e710.png)

Please see [this query](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/labels?q=size) from CAPV for a list of its labels and their descriptions. I will go ahead and precreate these labels in the VM Op repo.

Reply:
Okay, I [added the labels](https://github.com/vmware-tanzu/vm-operator/labels?q=size) to VM Op:

![image](https://user-images.githubusercontent.com/101085/215349182-78c3f3e3-785e-4d82-bcca-4839e17f71d5.png)

Reply:
Please update the LoC to match the numbers from the above labels that I created. Thanks!

Reply:
Thanks for adding these labels. I've updated the labeler config to match the above. I think we should remove the "ignoring generated files" as it's not supported yet?